app/src/main/java/com/teamagam/gimelgimel/app/message/viewModel/adapter/MessagesRecyclerViewAdapter.java

```diff
@@ -67,14 +74,44 @@ public synchronized void select(String messageId) {
         notifyDataSetChanged();
     }
 
+    private synchronized void selectWithEffect(MessageViewHolder viewHolder) {
```

Comment:
the name is confusing, it's name suggests that it selects the viewholder.
It's actually only animating the viewholder.
I'd name it "animateSelection"